src/supervisor/watchers/index.ts

```diff
@@ -60,9 +60,18 @@ export async function beginWatchingWorkloads(): Promise<void> {
       { namespace: config.WATCH_NAMESPACE },
       'kubernetes-monitor restricted to specific namespace',
     );
-    const namespaceResponse = await k8sApi.coreClient.readNamespace(
-      config.WATCH_NAMESPACE,
-    );
+    let namespaceResponse: any;
```

Comment:
why is this typed as any?

Reply:
because it can be either the correct response, or it can be empty. I have changed it now.